EIPS/eip-draft-returndata.md

```diff
@@ -0,0 +1,31 @@
+## Preamble
+
+    EIP: <to be assigned>
+    Title: Embedding transaction return data in receipts
```

Comment:
I guess the title should now be changed to `Embedding transaction status code in receipts`.